geode-core/src/distributedTest/java/org/apache/geode/cache30/DistributedNoAckRegionDUnitTest.java

```diff
@@ -220,17 +209,14 @@ public void run2() throws CacheException {
     vm0.invoke(new CacheSerializableRunnable("Put data") {
       @Override
       public void run2() throws CacheException {
-        LogWriterUtils.getLogWriter().info("testBug30705: starting initial data load");
-        Region region = getRootRegion().getSubregion(name);
-        final byte[] value = new byte[valueSize];
+        Region<Object, Object> region = getRootRegion().getSubregion(name);
+        final byte[] value = new byte[VALUE_SIZE_VM];
         Arrays.fill(value, (byte) 0x42);
-        for (int i = 0; i < numEntries; i++) {
-          if ((i % 1000) == 0) {
-            LogWriterUtils.getLogWriter().info("testBug30705: initial put #" + i);
+        for (int i = 0; i < NUM_ENTRIES_VM; i++) {
+          if ((i % 1000) != 0) {
+            region.put(i, value);
```

Comment:
I don't think this is the same as before.  Before it would still do the put even if the mod is 0.  It would just print out the log line if it happened to be the first.  I'm surprised if we don't get any test failures from this change.

Reply:
So you were right. I swapped the logic incorrectly.  The reason the test didn't fail is that it is ignored. I have swapped the logic back.

Reply:
The test was also no longer needed, so I deleted it.